src/gsi_ncdiag/gsi_ncdiag.py

```diff
@@ -1093,6 +1097,10 @@ def toIODAobs(self, OutDir, TotalBias=False, clobber=True, platforms=None):
                             #     tmp = hgt
                             outdata[(loc_mdata_name, 'MetaData')] = tmp
                             varAttrs[(loc_mdata_name, 'MetaData')]['units'] = 'm'
+                        elif v == 'bend':
+                            loc_mdata_name = 'impactHeightRO'
+                            outdata[(loc_mdata_name, 'MetaData')] = self.var(lvar)[idx]
+                            varAttrs[(loc_mdata_name, 'MetaData')]['units'] = 'm'
```

Comment:
here is the lovely mapping is `Height` is in the key and the variable is `bend` put that into `MetaData/impactHeightRO`

Reply:
@CoryMartin-NOAA was pretty obvious the `Height` was included later in the `all_LocKeyList` mapping

so since I'm not seeing `impact_height` in the `conv_gps` looks like this special case to map `Height` may be needed

Reply:
@BenjaminRuston, just want to add one thing, `Height` in `conv_gps` is computed as **impact_parameter - LOC**, not **impact_parameter - LOC - geoid**.